app/views/applications/show.html.erb

```diff
@@ -23,6 +25,11 @@
     <%= f.submit "Search" %>
     <% @pets.each do |pet| %>
       <p>Name: <%= pet.name %></p>
+      <%= form_with url: "/pet_applications", method: :post, local: true do |f| %>
```

Comment:
Big shoutout to cohortmates for helping get up over the hump here! We were so close with the button!! Knew what we wanted to do, just didn't know how to find those pesky params